waveform_editor/shape_editor/coil_currents.py

```diff
@@ -49,19 +49,13 @@ def create_ui(self, pf_active):
             coil_current = coil.current
             checkbox = pn.widgets.Checkbox(
                 margin=(30, 10, 10, 10),
-                disabled=self.nice_settings.param.mode.rx()
-                == self.nice_settings.DIRECT_MODE,
+                disabled=self.nice_settings.param.is_direct_mode.rx(),
```

Comment:
This can directly use the param, right?

```suggestion
                disabled=self.nice_settings.param.is_direct_mode,
```